molecule/rbac-mds-plain-custom-rhel/molecule.yml

```diff
@@ -256,15 +259,17 @@ provisioner:
           ldap.user.object.class: account
 
       cluster2:
+        redhat_java_package_name: java-11-openjdk
```

Comment:
how does this help?

Reply:
thanks for pointing out, I used it for testing and forgot to remove. Will do